bin/zopen-importenvs

```diff
@@ -1,18 +1,96 @@
-#!/bin/sh
-#
-# Used for development to source all the boot .envs
-# . importenvs
-
-if [ ! -d "${HOME}/zopen/boot/" ]; then
-  echo "boot directory does not exist.  Existing"
-  exit 4;
+#!/bin/env bash
+# bash is required because BASH_SOURCE is used here to locate the sourced script 
+# Used for development to source all the boot/prod .envs
+
+
+(return 0 2>/dev/null) && sourced=1 
+# What exit code did that give?
+if [[ $sourced -ne 1 ]]
+then
+    echo "This script is not sourced."
+    echo "USAGE: . ./zopen-importenvs <path to buildenv to fetch dependency>"
+    exit 6     
 fi
 
-for deps in $(find "${HOME}/zopen/boot/" -name ".env")
-do
-  echo "Found ${deps}"
-  depdir=$(dirname "${deps}")
-  cd $depdir
-  . ./.env
-  cd -
-done
+# below code fetches the ~/code/utils/bin path where the common.inc file is under its lib dir
+export bindir="$( cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd -P )"
+. "${bindir}/lib/common.inc"
```

Comment:
I wonder if we still need common.inc and if not, then perhaps we can make it a /bin/sh script rather than a bash script.  The reason we try to go with /bin/sh is because not all systems on z/OS have bash, but they all come with sh.